src/loggers/logger.service.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Injectable, OnModuleInit } from "@nestjs/common";
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  HttpStatus,
+} from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import {
+  Logger,
+  LoggerConfig,
+  LoggerMethods,
+} from "./interfaces/logger.interface";
+import * as fs from "fs";
+
+@Injectable()
+export class ScicatLogger implements OnModuleInit {
+  private loggers: {
+    logger: Logger;
+    methods?: LoggerMethods;
+  }[] = [];
+  constructor(private configService: ConfigService) {}
+
+  async onModuleInit() {
+    let loggerConfigs = this.configService.get<LoggerConfig[]>("loggerConfigs");
+
+    const filePath = "loggers.json";
+    if (fs.existsSync(filePath)) {
+      const data = fs.readFileSync(filePath, "utf8");
+      loggerConfigs = JSON.parse(data);
+    }
```

Comment:
I would move this section of code in the configuration controller. I would like to follow the concept that if you instantiate the configuration class, you have the full configuration and avoid loading portions of the configuration in other classes.
@bpedersen2 thoughts?